docs/src/inflation.md

```diff
@@ -0,0 +1,45 @@
+# Inflation 
+Inflation is a method which systematically adds stochastic perturbations to the parameter ensemble in the form of gaussian noise. 
+Following the standard Kalman update, the parameter ensemble is perturbed in the following manner:
+```math
+    u_{n+1} = u_n + \zeta_{n} \qquad (1)
+```
+
+where `` \zeta_{n} `` is noise drawn from a gaussian distribution. Two distinct forms of inflation are implemented.
+For both implementations, a scaling factor ``s`` is included to extend usage to cases with mini-batching. For mini-batching,
```

Comment:
could you first mention that ``Delta t``, then mention the `s` in regards to this?